src/components/content/content.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import './content.scss';
+import SurveyHeader from "./survey-header/survey-header";
+import SurveyTable from "./survey-table/survey-table";
+import SurveyTableHeader from "./survey-table/survey-table-header/survey-table-header";
+import SurveyTableRow from "./survey-table/survey-table-row/survey-table-row";
+import store from "../../store";
+import SurveyCheckbox from "../shared/survey-checkbox/survey-checkbox";
+import {checkAllSurveyTableRows} from "../../store/actions";
+
+
+class ContentComponent extends React.Component {
+    state = {
+        headers: [],
+        sorted: {},
+        surveys: []
+    };
+
+    checkAll = false;
+
+    componentDidMount() {
+        this.setState(store.getState());
```

Comment:
It's not common directly access the redux store and subsribe to it, there is a library called `react-redux`, which has a `connect` function to create container components. Same applies for the actions, too. You could read about this [here](https://redux.js.org/basics/usage-with-react).

Reply:
I have used redux only because it was mentioned in a task. I really don't see any purpose to use it in this small apps, as well as a react-router.